Core/include/Managers/AudioManager.h

```diff
@@ -10,6 +10,7 @@
 #include <Components/TransformComponent.h>
 
 class AudioManager : public System {
+friend class AudioComponent;
```

Comment:
Icky use of friend class, it shouldnt be hard to conform to OOP principles with the audio classes given their existing design

Reply:
Can remove now. Forgot this was there, it was deprecated in favour of making a conversion function static, as it should have been.